wire-library/wire-tests/src/jvmKotlinInteropTest/kotlin/com/squareup/wire/SerializableTest.kt

```diff
@@ -58,7 +58,7 @@ class SerializableTest {
     val goldenSerialized = ("rO0ABXNyACdjb20uc3F1YXJldXAud2lyZS5NZXNzYWdlU2VyaWFsaXplZEZvcm0AAAAA" +
         "AAAAAAIAAlsABWJ5dGVzdAACW0JMAAxtZXNzYWdlQ2xhc3N0ABFMamF2YS9sYW5nL0NsYXNzO3hwdXIAAltCrPMX" +
         "+AYIVOACAAB4cAAAAAkoY1IFdGFjb3N2cgA0Y29tLnNxdWFyZXVwLndpcmUucHJvdG9zLmtvdGxpbi5zaW1wbGUu" +
-        "U2ltcGxlTWVzc2FnZbGI6djBY2gnAgAMSQAOcmVxdWlyZWRfaW50MzJMABRkZWZhdWx0X2ZvcmVpZ25fZW51bXQA" +
+        "U2ltcGxlTWVzc2FnZavdiLU6MxJkAgAMSQAOcmVxdWlyZWRfaW50MzJMABRkZWZhdWx0X2ZvcmVpZ25fZW51bXQA" +
```

Comment:
What happened here?

Reply:
Deserialization failed, the class file has likely changed due to the bump to Kotlin 1.4.